src/pages/inventory/tabs/DefaultTabTheme.js

```diff
@@ -48,6 +48,17 @@ export const customTheme = {
           color: '#000000',
           borderTop: '6px solid #9664C7',
         },
+        '&.treatment': {
```

Comment:
The tab colors needs to be updated to match the original ordering of the colors. (Blue, yellow, minty green, purple, dark green, burgundy)

Reply:
Resolved.